packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleSelectableGrid.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+  Flex,
+  Image,
+  Text,
+  Card,
+  cardSelectableBehavior,
+  Grid,
+  cardsContainerBehavior,
+  Checkbox,
+  Button,
+  screenReaderContainerStyles,
+  hiddenComponentBehavior,
+} from '@fluentui/react-northstar';
+import * as React from 'react';
+import * as _ from 'lodash';
+
+type SelectableCardProps = {
+  index?: number;
+  title?: string;
+  selected?: boolean;
+  handleClick?: Function;
+};
+
+const SelectableCard: React.FC<SelectableCardProps> = ({ title, index, selected, handleClick, ...unhadledProps }) => {
+  const selectedMessageId = `selectedMessageId${index}`;
+  const selectedMessage = 'selected';
+  const notSelectedMessage = 'not selected';
+  return (
+    <Card
+      id={`card${index}`}
+      accessibility={cardSelectableBehavior}
+      aria-labelledby={`card${index} ${selectedMessageId}`}
+      aria-roledescription="user card"
+      onClick={() => {
+        handleClick(!selected, index);
+      }}
+      selected={selected}
+      {...unhadledProps}
+    >
+      <Card.Header>
+        <Text content={title} weight="bold" />
+      </Card.Header>
+      <Card.TopControls>
+        <Checkbox
+          accessibility={hiddenComponentBehavior}
+          checked={selected}
+          onClick={(event, props) => {
+            event.preventDefault();
+            handleClick(props.checked, index);
+          }}
+        />
+        <div id={selectedMessageId} style={screenReaderContainerStyles} aria-live="polite" role="presentation">
+          {selected ? selectedMessage : notSelectedMessage}
+        </div>
+      </Card.TopControls>
+
+      <Card.Body>
+        <Flex column gap="gap.small">
+          <Image src="public/images/wireframe/square-image.png" />
+        </Flex>
+      </Card.Body>
+    </Card>
+  );
+};
+
+interface SelectableCardsGridAction {
+  type: 'TOGGLE_ITEM' | 'TOGGLE_ALL';
+  selected: boolean;
+  itemKey?: string;
+}
+
+interface SelectableCardsGridState {
+  cards: Record<string, boolean>;
+}
+
+const selectableCardsGridStateReducer: React.Reducer<SelectableCardsGridState, SelectableCardsGridAction> = (
+  state,
+  action,
+) => {
+  switch (action.type) {
+    case 'TOGGLE_ITEM': {
+      return { cards: { ...state.cards, [action.itemKey]: action.selected } };
```

Comment:
```suggestion
      return { cards: { ...state.cards, [action.index]: action.selected } };
```

Why it's `itemKey`? Should be just `index`?